code/modules/antagonists/creep/creep.dm

```diff
@@ -35,7 +35,7 @@
 	var/icon/final_icon = finish_preview_icon(obsessed_icon)
 
 	final_icon.Blend(
-	icon('icons/ui_icons/antags/obsessed.dmi', "obsession"),
+	icon('icons/UI_Icons/antags/obsessed.dmi', "obsession"),
```

Comment:
doesn't the apprentice one also use this folder or did i put it in the correct one

Reply:
i did.

Reply:
I searched all instances and this was the only one not using proper case
Though this is dumb and it should all be lowercase, it's too late to change now without breaking shit or renaming the folder entirely to something off of ui_icons.